galeracluster/source/documentation/install-mysql.rst

```diff
@@ -224,25 +228,13 @@ For Galera Cluster 8.4:
 
    apt-get install galera-4 galera-arbitrator-4 mysql-wsrep-8.4
```

Comment:
please do not remove 8.. for 8.0 the 8.0 insturctions must exist. for 8.4 this is ok

Reply:
Fixed